_includes/activity/intro-to-github/04-create-a-file-on-github.md

```diff
@@ -1,17 +1,14 @@
-{% capture activity-text %}
-# Activity: Adding a New Pin to the Map
+It's time to add your file to the repository:
 
 1. In your new branch, add a new file named: `_pins/YOUR-USERNAME.json`.
-1. [Click here](http://mynasadata.larc.nasa.gov/latitudelongitude-finder/) to identify your latitude and longitude.
-1. Enter the following code in your file:
-![]({{site.baseurl}}/images/activity-content-intro.png){: .align-center}
-1. Ensure you've replaced: `YOUR_USERNAME` with your GitHub username, `YOUR_LATITUDE` with your north/south coordinate, and `YOUR_LONGITUDE` with your east/west coordinate.
-1. Enter a commit message that describes your addition.
-1. Commit your change to your branch.
+1. Enter the following content into your file:
 
+        ---
+        githubHandle: YOUR-USERNAME
+        latitude: YOUR_LATITUDE
+        longitude: YOUR_LONGITUDE
+        ---
```

Comment:
@hectorsector This is entirely @beardofedu and @brianamarie 's work from Pages. 👍 

I'm pretty thrilled about it, too.